gateway/1.8/modules/ROOT/pages/flex-conn-third-party-logs-config.adoc

```diff
@@ -39,7 +39,7 @@ To configure access logs in your log output, you must apply a Message Logging po
 
 To configure a Message Logging policy, see xref:flex-conn-message-log.adoc[].
 
-Configuring a Message Logging policy makes access logs visible in both API Manager *Logs* and your third-party log services.
+Configuring a Message Logging policy makes access logs visible in API Manager, Runtime Manager, and your third-party log services.
```

Comment:
Should API Manager still be mentioned? You removed it from other pages. If customers can still view logs in API Manager then it should be included in the other pages.